modules/cache/cache.ts

```diff
@@ -53,8 +64,15 @@ export const getCacheInfo = async (name: string, network: SupportedNetworks): Pr
     const path = getFilePath(name, currentNetwork);
 
     if (isRedisCache && redis) {
-      const ttl = await redis?.ttl(path);
-      return ttl;
+      // if proposals, there are probably multiple keys, get the ttl for first one
```

Comment:
What I meant to say is that if we're fetching `proposals` from the cache, multiple proposals keys are likely cached because they get cached with query params in the key name.

I'll update it 👍